griptape/drivers/prompt/base_prompt_driver.py

```diff
@@ -65,6 +66,7 @@ def after_run(self, result: Message) -> None:
                 ),
             )
 
+    @observable(tags=["PromptDriver.run()"])
```

Comment:
Why do we need this tag here but not the other places?

Reply:
This is so we can group spans for all subclasses. The span names are "ClassName.MethodName()", but it doesn't give an easy way to say, all of these spans are for any prompt driver without keeping a list of all prompt driver class names.

Does that make sense?

Reply:
Got it, thanks!

Reply:
Should we add this for other top-level Drivers like Embedding Drivers?

Reply:
We can, and will probably eventually want to, but I was thinking we could wait until a real use case arrives. Once we get to play with the data a little more we'll have a better idea of what is useful and what just adds noise.